zuul.d/edpm.yaml

```diff
@@ -58,3 +58,5 @@
       - ^ci/playbooks/edpm_baremetal_deployment/run.yml
       - ^deploy-edpm.yml
       - ^scenarios/centos-9/edpm_baremetal_deployment_ci.yml
+      - ^plugins/action/ci_script.py
```

Comment:
@cjeanner actually this one is wrong... this job is not in the baremetal template (i.e. what i added to projects.yaml).

the cifmw-crc-podified-edpm-baremetal Is in the in the template and is now running here but i think this bit is irrelevant unless we add this job to that template.

or wire it up to run otherwise. any opinion? leaving for now and if it merges can tidy up on the next patch that will remove the ci_make

Reply:
fixed elsewhere in latest push 
marking resolved